assemblers/org.osbuild.qemu

```diff
@@ -364,12 +377,14 @@ def filesystem_from_json(js) -> Filesystem:
 
 
 def partition_from_json(js) -> Partition:
-    p = Partition(pttype=js.get("type"),
-                  start=js.get("start"),
-                  size=js.get("size"),
-                  bootable=js.get("bootable"),
-                  name=js.get("name"),
-                  uuid=js.get("uuid"))
+    p = Partition(
+        pttype=js.get("type"),
```

Comment:
The `Partition` gets lost by being on the same indent as the arguments.

Reply:
Accidental `p = ` being the same length as an indent. Could be fixed by naming it `part` or `partition` luckily.